packages/alfa-rules/src/common/act/question.ts

```diff
@@ -18,6 +18,7 @@ export namespace Question {
     "node[]": Iterable<Node>;
```

Comment:
There is an unused import in this file (`Diagnostic`)

Reply:
Was cleaned in https://github.com/Siteimprove/alfa/pull/1115/commits/392f5461dee6e6b9045097464cf5b49d306ef734